2025-07-19 11:08:42,549 INFO    MainThread:2269 [wandb_init.py:setup_run_log_directory():703] Logging user logs to /jupyter-tutorial/AAIPL_134_199_198_15/wandb/run-20250719_110842-nzwfl04c/logs/debug.log
2025-07-19 11:08:42,549 INFO    MainThread:2269 [wandb_init.py:setup_run_log_directory():704] Logging internal logs to /jupyter-tutorial/AAIPL_134_199_198_15/wandb/run-20250719_110842-nzwfl04c/logs/debug-internal.log
2025-07-19 11:08:42,549 INFO    MainThread:2269 [wandb_init.py:init():831] calling init triggers
2025-07-19 11:08:42,549 INFO    MainThread:2269 [wandb_init.py:init():836] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-07-19 11:08:42,549 INFO    MainThread:2269 [wandb_init.py:init():872] starting backend
2025-07-19 11:08:42,549 INFO    MainThread:2269 [wandb_init.py:init():875] sending inform_init request
2025-07-19 11:08:42,550 INFO    MainThread:2269 [wandb_init.py:init():883] backend started and connected
2025-07-19 11:08:42,552 INFO    MainThread:2269 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2025-07-19 11:08:42,552 INFO    MainThread:2269 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2025-07-19 11:08:42,552 INFO    MainThread:2269 [wandb_init.py:init():956] updated telemetry
2025-07-19 11:08:42,553 INFO    MainThread:2269 [wandb_init.py:init():980] communicating run to backend with 90.0 second timeout
